lib/nc_time_axis/tests/__init__.py

```diff
@@ -0,0 +1,19 @@
+# (C) British Crown Copyright 2016, Met Office
+#
+# This file is part of nc-time-axis.
+#
+# nc-time-axis is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
```

Comment:
These headings are confusing. The code is BSD-3 so best just to not include the year headings at all.